x-pack/legacy/plugins/siem/server/lib/detection_engine/types.ts

```diff
@@ -44,20 +45,30 @@ export interface RuleAlertParams {
   to: string;
   threats: ThreatParams[] | undefined | null;
   type: 'query' | 'saved_query';
+  updatedAt: string | undefined | null;
```

Comment:
I'm wondering if we want these to be nullable at all or if we want to enforce these are always here for both of these. I would think they are always going to be available and never optional with `undefined | null`